lib/connect-session-sequelize.js

```diff
@@ -29,7 +29,10 @@ module.exports = function SequelizeSessionInit(Store) {
     }
 
     var key;
-    this.options = defaultOptions;
+    this.options = {};
+    for (key in defaultOptions) {
+      this.options[key] = defaultOptions[key];
+    }
```

Comment:
Because otherwise the `defaultOptions` object would be shared between instances as `this.options`.Should someone, for whatever reason, use two instances of `SequelizeStore`, they wouldn't be able to specify two different `extendDefaultFields` functions.

Now that I think of it, I didn't notice there was a check below for `if key !== 'db'`. I will push an update which uses only a single iteration over `defautObjects`.